src/clientsettingsdlg.cpp

```diff
@@ -132,7 +133,7 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
 
     cbxSoundcard->setAccessibleName ( tr ( "Sound card device selector combo box" ) );
 
-#if defined( _WIN32 ) && !defined( JACK_REPLACES_ASIO )
+#    if defined( _WIN32 )
     // set Windows specific tool tip
```

Comment:
These commects explicitly mention ASIO - so you should leave in the JACK_REPLACES_ASIO, I think?